sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -409,15 +492,17 @@ abstract class AbstractSync(
         imsProject: IMSProject,
         finalBlock: List<TimelineItem>,
         relevantTimeline: List<TimelineItem>,
-        restoresDefaultState: Boolean
+        restoresDefaultState: Boolean,
+        virtualIDs: Map<TimelineItem, String>
```

Comment:
docs for new param

Reply:
fafafc8c6b5332803aa70add51f61e003561ff7d